### PR TITLE
Geomap: Update whats new docs for 9.1

### LINKED ISSUE #53629: Geomap: Update What's New Section

### PR DESCRIPTION
**Pull request by nmarrs**
**What this PR does / why we need it**:
Include what's new info for geomap panel for 9.1 release

Note: dependent on #53552 being merged (hence the `blocked` label)

**Which issue(s) this PR fixes**:

Fixes #53629

### CODE REVIEW

File: docs/sources/whatsnew/whats-new-in-v9-1.md [hunk 1]
```diff
@@ -86,6 +86,13 @@ To learn more about the heatmap panel, see the [documentation]({{< relref "../vi
 
 {{< figure src="/static/img/docs/panels/heatmap-panel-9-1.png" max-width="750px" caption="A heatmap panel" >}}
 
+#### Geomap
+
+Geomap now includes a new [measuring tool]({{< relref "../visualizations/geomap/controls/#show-measure-tools" >}}) that allows you to measure both distances and areas.
```
Comment (gguillotte-grafana) [inline docs/sources/whatsnew/whats-new-in-v9-1.md:91]:
```suggestion
You can now measure both distances and areas on Geomap visualizations by using the panel's new [measure tools]({{< relref "../visualizations/geomap/controls/#show-measure-tools" >}}).
```

File: docs/sources/whatsnew/whats-new-in-v9-1.md [hunk 2]
```diff
@@ -86,6 +86,13 @@ To learn more about the heatmap panel, see the [documentation]({{< relref "../vi
...
+To learn more about the geomap panel, see the [documentation]({{< relref "../visualizations/geomap/" >}}).
```
Comment (gguillotte-grafana) [inline docs/sources/whatsnew/whats-new-in-v9-1.md:92]:
```suggestion
To learn more about the Geomap panel, see the [documentation]({{< relref "../visualizations/geomap/" >}}).
```

File: docs/sources/whatsnew/whats-new-in-v9-1.md [hunk 3]
```diff
@@ -86,6 +86,13 @@ To learn more about the heatmap panel, see the [documentation]({{< relref "../vi
...
+
+{{< figure src="/static/img/docs/geomap-panel/geomap-map-controls-measure-area-9-1-0.png" max-width="750px" caption="Geomap panel measuring tool - area" >}}
```
Comment (gguillotte-grafana) [inline docs/sources/whatsnew/whats-new-in-v9-1.md:94]: No action necessary, commenting for posterity: this image is added in grafana/website#9708.

**Review by gguillotte-grafana — CHANGES_REQUESTED** [review]

**Review by gguillotte-grafana — APPROVED** [review]
Suggested some changes, but I don't want to block merging on them.

**Review by drew08t — APPROVED** [review]

**Pull request merged**